.changeset/honest-queens-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
```

Comment:
`patch`

Reply:
changed it to patch , as no changeset create an warning